frontend/src/assets/main.scss

```diff
@@ -228,6 +228,12 @@ a button:focus-visible {
     }
   }
 
+  table tbody,
+  table tbody tr td:focus-visible{
+    outline: 0 none;
+    box-shadow: inset 0px 0px 0 0.1rem #a6d5fa;
```

Comment:
we used a width or 0.2rem for the anchors. (see line 83).
maybe this should also be: `box-shadow: 0 0 0 0.2rem #a6d5fa;` or we make them all 0.1rem